openpype/hosts/blender/api/utils.py

```diff
@@ -372,6 +372,10 @@ def get_datablocks_with_filepath(
                 and not datablock.filepath == ""
                 and not datablock.library
                 and not datablock.is_library_indirect
+                and (
+                    not isinstance(datablock, bpy.types.Library)
+                    or not datablock.parent
```

Comment:
if datablock has no attribute `parent` it'll raise an error.

I think this will do the same result and be more safer regarding this attribute `and not (isinstance(datablock, bpy.types.Library) and datablock.parent)`